src/mahoji/commands/runecraft.ts

```diff
@@ -122,46 +134,55 @@ export const runecraftCommand: OSBMahojiCommand = {
 
 		let { tripLength } = runeObj;
 		const boosts = [];
-		if (userHasGracefulEquipped(user)) {
-			tripLength -= tripLength * 0.1;
-			boosts.push('10% for Graceful');
+		if (user.isIronman && runners) {
+			runners = false;
+			boosts.push('Ironmen stand on their own');
```

Comment:
Not really necessary to push it to boosts and doesnt really explain why this message is present.

I understand that its because it involves trading but maybe more useful to say "Ironmen can't utilize runners." and not include it in the boost string or throw a generic return like most Ironman clauses.

Reply:
fixed